.github/update.yml

```diff
@@ -0,0 +1,36 @@
+name: "Update generated files"
+# This rule is not run automatically.
```

Comment:
remove comments?

Reply:
It just describes how the tool is run copied directly from the template (https://github.com/martinthomson/internet-draft-template/blob/main/.github/workflows/update.yml#L2)